src/Files.App/Helpers/FilePropertiesHelpers.cs

```diff
@@ -15,11 +15,14 @@
 
 namespace Files.App.Helpers
 {
-	public static class FilePropertiesHelpers
+	public class FilePropertiesHelpers
```

Comment:
Why has the class been made non-static? All the methods inside are `static`.

If the class has to stay non-static, its name should be `FilePropertiesHelper` (without an 's').

This is for me the biggest question mark on this PR.

Reply:
This class was already static but became non-static yesterday (#10283), although all its content is static. I reapplied the changes from this commit which broke my merge. It actually looks like the class should be static. @lukeblevins, Can you explain why this class is no longer static? It is never instantiated.

Reply:
This could have been a small mistake with the merge conflict resolution from that pull request. This can probably be changed back safely.

Reply:
The class is now static.